packages/app/cypress/e2e/support/execute-spec.ts

```diff
@@ -23,13 +22,10 @@ declare global {
   }
 }
 
-// Here we export the function with no intention to import it
-// This only tells the typescript type checker that this definitely is a module
-// This way, we are allowed to use the global namespace declaration
-export const waitForSpecToFinish = (expectedResults, timeout?: number) => {
+const waitForSpecToFinish = (expectedResults, timeout?: number) => {
   // First ensure the test is loaded
-  cy.get('.passed > .num').should('contain', '--')
```

Comment:
Race condition - sometimes the rest actually executes fast enough, and it's not `--`, but the number of passes. So, better just verify the elements exist.